app/modules/sightings/resources.py

```diff
@@ -259,6 +272,18 @@ def post(self, args):
                 'Sighting.post imbalanced encounters in %r or %r'
                 % (request_in, result_data),
             )
+
+        enc_anns = []  # list of lists of annotations for each encounter (if applicable)
```

Comment:
if _get_annotations was changed to take request_in['encounters'] you could put the for loop in there